databases/week2/week2exercise2.sql

```diff
@@ -0,0 +1,29 @@
+-- Create a new database containing the following tables:
+-- Class: with the columns: id, name, begins (date), ends (date)
+-- Student: with the columns: id, name, email, phone, class_id (foreign key)
+CREATE DATABASE school;
+USE school;
+CREATE TABLE `class` (
+`id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+`name` varchar(255) NOT NULL,
+`begins (date)` DATETIME NOT NULL DEFAULT NOW(),
+`ends (date)` DATETIME NOT NULL DEFAULT NOW(),
```

Comment:
I would not recommend putting spaces and parenthesis in column names. I would just call the two DATETIME columns `begins` and `ends`

Reply:
I updated the file with columns begins and ends